src/data/chartsForIsland.json

```diff
@@ -0,0 +1,51 @@
+[
+  "Chart for Forsaken Fortress",
+  "Chart for Star Island",
+  "Chart for Northern Fairy Island",
+  "Chart for Gale Isle",
+  "Chart for Crescent Moon Island",
+  "Chart for Seven-Star Isles",
+  "Chart for Overlook Island",
+  "Chart for Four-Eye Reef",
+  "Chart for Mother and Child Isles",
+  "Chart for Spectacle Island",
+  "Chart for Windfall Island",
+  "Chart for Pawprint Isle",
+  "Chart for Dragon Roost Island",
+  "Chart for Flight Control Platform",
+  "Chart for Western Fairy Island",
+  "Chart for Rock Spire Isle",
+  "Chart for Tingle Island",
+  "Chart for Northern Triangle Island",
+  "Chart for Eastern Fairy Island",
+  "Chart for Fire Mountain",
+  "Chart for Star Belt Archipelago",
+  "Chart for Three-Eye Reef",
+  "Chart for Greatfish Isle",
+  "Chart for Cyclops Reef",
+  "Chart for Six-Eye Reef",
+  "Chart for Tower of the Gods",
+  "Chart for Eastern Triangle Island",
+  "Chart for Thorned Fairy Island",
+  "Chart for Needle Rock Isle",
+  "Chart for Islet of Steel",
+  "Chart for Stone Watcher Island",
+  "Chart for Southern Triangle Island",
+  "Chart for Private Oasis",
+  "Chart for Bomb Island",
+  "Chart for Bird's Peak Rock",
+  "Chart for Diamond Steppe Island",
+  "Chart for Five-Eye Reef",
+  "Chart for Shark Island",
+  "Chart for Southern Fairy Island",
+  "Chart for Ice Ring Isle",
+  "Chart for Forest Haven",
+  "Chart for Cliff Plateau Isles",
+  "Chart for Horseshoe Island",
+  "Chart for Outset Island",
+  "Chart for Headstone Island",
+  "Chart for Two-Eye Reef",
+  "Chart for Angular Isles",
+  "Chart for Boating Course",
+  "Chart for Five-Star Isles"
```

Comment:
We don't need this JSON file. Instead, we should take the list of islands and then call a function in `LogicHelper` to convert those island names to `Chart for ${island}`.